### PR TITLE
Move user system functions and add partial support for swagger

### PR DESCRIPTION
**Pull request by BarVolunteering**
Fully resolving issue #2000 and partly resolving issue #1903, A full resolution for #1903 will be implemented depending on the result of this PR.

### CODE REVIEW

File: anyway/flask_app.py [hunk 1]
```diff
@@ -2050,3 +1418,90 @@ def add_user_to_org() -> Response:
 #     return Response(json_data, mimetype="application/json")
 #
 #
+
+# User system API
+@login_manager.user_loader
```
Comment (ziv17) [inline anyway/flask_app.py:1423]: I do not know what are these annotations `@login_manager`, `@identity_loaded`, etc. If they are not HTTP endpoint URLs, maybe it is an opportunity to move them out of `flask_app.py`?

**Review by ziv17 — COMMENTED** [review]
Good work!

I see you started to add Swagger, Nice!

One question below regarding some annotations.

Regarding directories structure. It is not actually related to this PR, I write it here due to the addition of `user_system`
- I think Infographics, is a view, so maybe it should go undr `views` directory.
- Is `user_system` a view?

File: anyway/flask_app.py (hunk 1 above)
Comment (BarVolunteering) [reply anyway/flask_app.py:1423]: I will move those functions out of `flask_app.py`

**Comment by BarVolunteering** [comment]
@ziv17 I am not sure if to call `user_system` a view, maybe more of a subsystem.
Moving the Infographics under the `views` directory is a good idea, maybe we should talk about it in the next meeting?

**Review by ziv17 — APPROVED** [review]
Well done!

**Pull request merged**